src/components/confirm-dialog/ConfirmDialog.tsx

```diff
@@ -41,15 +45,15 @@ const ConfirmDialog: FC<ConfirmDialogProps> = ({
       <IconButton onClick={onDismiss} sx={styles.icon}>
         <CloseIcon />
       </IconButton>
-      <DialogContent dividers sx={styles.content}>
+      <DialogContent sx={styles.content}>
         <Typography variant='subtitle1'>{t(message)}</Typography>
       </DialogContent>
       <DialogActions sx={styles.actions}>
         <Button onClick={onConfirm} size='large' variant='tonal'>
-          {t('common.yes')}
+          {secondaryBtnTitle ? secondaryBtnTitle : t('common.yes')}
         </Button>
         <Button onClick={onDismiss} size='large' variant='contained'>
-          {t('common.no')}
+          {mainBtnTitle ? mainBtnTitle : t('common.no')}
```

Comment:
I guess, you can set common.no and common.yes as default values in props

Reply:
done